src/main/java/com/opinta/service/CounterpartyService.java

```diff
@@ -18,7 +18,7 @@
 
     Counterparty getEntityByUser(User user) throws IncorrectInputDataException;
     
-    List<Counterparty> getEntityByPostcodePoolUuid(UUID postcodePoolUuid) throws IncorrectInputDataException;
+    List<Counterparty> getEntitiesBySharedPostcodePoolUuid(UUID postcodePoolUuid) throws IncorrectInputDataException;
```

Comment:
Plz, rename to something like this: getAllEntitiesByPostcodePoolUuid
Plz, refer to the existent methods of the same functionalities